core/src/main/java/feast/core/job/JobUpdateTask.java

```diff
@@ -195,13 +231,9 @@ private Job updateStatus(Job job) {
     return job;
   }
 
-  String createJobId(String sourceId, String storeName) {
+  String createJobId(Source source, String storeName) {
     String dateSuffix = String.valueOf(Instant.now().toEpochMilli());
-    String[] sourceParts = sourceId.split("/", 2);
-    String sourceType = sourceParts[0].toLowerCase();
-    String sourceHash =
-        Hashing.murmur3_128().hashUnencodedChars(sourceParts[1]).toString().substring(0, 10);
-    String jobId = String.format("%s-%s-to-%s-%s", sourceType, sourceHash, storeName, dateSuffix);
+    String jobId = String.format("%s-to-%s-%s", store, storeName, dateSuffix);
```

Comment:
probably it was meant `"%s-to-%s-%s", source`

Reply:
Coorrected